src/tlo/simulation.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, *, start_date: Date, seed: int = None, log_config: dict = Non
         self.rng = np.random.RandomState(np.random.MT19937(self._seed_seq))
 
     def configure_logging(self, filename: str = None, directory: Union[Path, str] = "./outputs",
-                          custom_levels: Dict[str, int] = None, suppress_stdout: bool = False):
+                          custom_levels: Dict[str, int] = None, suppress_stdout: bool = True):
```

Comment:
I think we can also do this in analyses file
```suggestion
                          custom_levels: Dict[str, int] = None, suppress_stdout: bool = False):
```

Reply:
yes, right, definitely don;t want to update the core framework